content/docs/bdd/_index.md

```diff
@@ -6,4 +6,107 @@ menu: main
 iconClass: fas fa-comments
```

Comment:
What does this front-matter tag do?

Reply:
I think it adds an icon for this section on the main nav on the left?